docs/odh-rhods/custom-runtime-triton.md

```diff
@@ -0,0 +1,227 @@
+# Deploying and using a Custom Serving Runtime in RHODS
+
```

Comment:
I would modify the title to RHODS/ODH and add a quick note that the procedure, although described for RHODS, works the same for both.

Reply:
done in https://github.com/opendatahub-io-contrib/ai-on-openshift/pull/38/commits/0a759ddaa4597ddf7da23a67fd19537a4e4acf60